api/Path2D.json

```diff
@@ -147,6 +147,438 @@
             "deprecated": false
           }
         }
+      },
+      "arc": {
+        "__compat": {
+          "spec_url": "https://html.spec.whatwg.org/multipage/canvas.html#dom-context-2d-arc-dev",
+          "support": {
+            "chrome": {
+              "version_added": "36"
+            },
+            "chrome_android": {
+              "version_added": "36"
+            },
+            "edge": {
+              "version_added": "14"
+            },
+            "firefox": {
+              "version_added": "31"
+            },
+            "firefox_android": {
+              "version_added": "31"
+            },
+            "ie": {
+              "version_added": false
+            },
+            "opera": {
+              "version_added": "23"
+            },
+            "opera_android": {
+              "version_added": "24"
+            },
+            "safari": {
+              "version_added": "8"
```

Comment:
It's weird that none of the members of the interface are marked as supported in Safari 7 but the interface itself is. Can you check if that's really correct?

Reply:
It appears that I was the one who set this to Safari 7.  This was probably an issue due to the confusion regarding Safari 7.1 (it was in #7950, which was last year, ergo we had no idea Safari 7.1 was an 8.0 backport).  I'll update this in a quick follow-up PR!

Reply:
Ah, I see, thanks!